docs/guides/admin/docs/index.md

```diff
@@ -28,19 +28,17 @@ The Opencast Release Documentation is the official Opencast documentation for ea
     * [HTTPS Configuration](configuration/https/index.md)
     * [Workflow Configuration](configuration/workflow.md)
     * [Encoding Configuration](configuration/encoding.md)
+    * [Plugin Management](configuration/plugin-management.md)
+    * [Atom and RSS Feed](configuration/atomrss.md)
+    * [AWS S3 Distribution](configuration/awss3distribution.md)
+    * [Media Module](configuration/mediamodule.configuration.md)
+    * [Metrics (OpenMetrics, Prometheus)](configuration/metrics.md)
+    * [New Admin UI](configuration/admin-ui/new-admin-ui.md)
+    * [Stream Security](configuration/stream-security/stream-security-overview.md)
+    * [Stand-alone Video Editor](configuration/videoeditor.frontend.md)
+    * [Studio](configuration/studio.md)
+    * [Text Extraction](configuration/textextraction.md)
+    * [Video Segmentation](configuration/videosegmentation.md)
+    * [YouTube Publication](configuration/youtubepublication.md)
     * [more...](configuration/index.md)
```

Comment:
I don't dislike this as much as I thought I would, but I would definitely edit the list of default config items shown to shorten it up.

What about this?

```suggestion
    * [Plugin Management](configuration/plugin-management.md)
    * [Atom and RSS Feed](configuration/atomrss.md)
    * [Metrics (OpenMetrics, Prometheus)](configuration/metrics.md)
    * [New Admin UI](configuration/admin-ui/new-admin-ui.md)
    * [Stream Security](configuration/stream-security/stream-security-overview.md)
    * [Stand-alone Video Editor](configuration/videoeditor.frontend.md)
    * [Studio](configuration/studio.md)
    * [more...](configuration/index.md)
```

(drops s3 dist, media module, text extraction, video segmentation, and youtube publication).  I could see arguments for removing other things (admin ui, feeds, metrics) too, but this at least shortens the list down to keep the initial shock of the list size down.

Reply:
I have no opinion on what is and is not shown in that list (it always seemed somewhat superfluous to me), so I'll go along with your suggestion if no one disagrees.